pkg/storage/storagewrappers/contextualtuplereader.go

```diff
@@ -8,18 +8,18 @@ import (
 	"github.com/openfga/openfga/pkg/tuple"
 )
 
-// NewCombinedTupleReader returns a TupleReader that reads from a persistent datastore and from the contextual
+// NewContextualTupleReader returns a TupleReader that reads from a persistent datastore and from the contextual
 // tuples specified in the request
-func NewCombinedTupleReader(ds storage.RelationshipTupleReader, contextualTuples []*openfgav1.TupleKey) storage.RelationshipTupleReader {
-	return &combinedTupleReader{RelationshipTupleReader: ds, contextualTuples: contextualTuples}
+func NewContextualTupleReader(ds storage.RelationshipTupleReader, contextualTuples []*openfgav1.TupleKey) *ContextualTupleReader {
+	return &ContextualTupleReader{RelationshipTupleReader: ds, contextualTuples: contextualTuples}
 }
 
-type combinedTupleReader struct {
+type ContextualTupleReader struct {
```

Comment:
```suggestion
type contextualTupleReader struct {
```

This doesn't need to be exported. We want to encourage upstream to depend on the interface `storage.RelationshipTupleReader` instead of this directly.

Reply:
I need it to be exported so that I can reference it explicitly as function parameters within `check`: https://github.com/openfga/openfga/pull/948/files#diff-5074b7eeccca5be19ff0fa2a8f712335a6ed075628b66a46697ca9d8ca99fbe0R765